resource_management/Rakefile

```diff
@@ -106,102 +112,107 @@ def each_gem
   end
 end
 
+version = File.read(File.expand_path('../ARM_VERSION', __FILE__)).strip
 REGEN_METADATA = {
     autorest_loc: '../../../autorest/binaries/net45/AutoRest.exe',
+    azure_arm: {
+        version: version,
+        tag: 'azure_arm'
+    },
     azure_mgmt_authorization: {
         spec_uri: 'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-authorization/2015-07-01/swagger/authorization.json',
         ns: 'Azure::ARM::Authorization',
-        version: '0.2.1',
+        version: version,
```

Comment:
could this be Azure::ARM::Authorization::VERSION? I'm not sure if all gems will always rev at the same time. It's fine for the current case, so we may end up modifying this later anyway.